index.html

```diff
@@ -4,9 +4,167 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
     <title>Douglas' Portifolio page</title>
 </head>
 <body>
-    <h2>Portifolio</h2>
+<div class="main-layout">
+
+    <nav>
+        <span>Luzinda Douglas</span> 
+        <img src="https://img.icons8.com/ios-glyphs/30/ffffff/menu--v1.png" alt="Hamburger menu icon">  
+    </nav>  
```

Comment:
Kindly use flexbox to make the design of the header responsive and remove the white space around it  as shown in the comparison 🤗

Your design:
![image](https://user-images.githubusercontent.com/73714615/186861687-f84de8b9-f77c-4f8b-8f5f-5d2a1c442dc4.png)

Figma Design:
![image](https://user-images.githubusercontent.com/73714615/186861815-b1c00775-5920-4ccb-bfef-abca0c58fdf4.png)